libtransmission/torrent.h

```diff
@@ -769,6 +769,24 @@ struct tr_torrent
         return sequential_download_;
     }
 
+    bool set_sequential_download_from_piece(tr_piece_index_t piece) noexcept
+    {
+        auto const is_valid = piece < piece_count();
+        TR_ASSERT(is_valid);
+        if (is_valid && piece != sequential_download_from_piece_)
+        {
+            sequential_download_from_piece_ = piece;
+            sequential_download_from_piece_changed_.emit(this, piece);
+            return true;
+        }
+        return false;
```

Comment:
@tearfur I applied the changes, but I'm not sure why we return a bool here as we don't use the returned value ?

Reply:
No worries we can keep it like this :+1: